health-services/project/src/main/java/org/egov/project/repository/rowmapper/LocationCaptureRowMapper.java

```diff
@@ -0,0 +1,91 @@
+package org.egov.project.repository.rowmapper;
+
+import java.sql.ResultSet;
```

Comment:
I think user action rowmapper will serve the same use case, remove this.

Reply:
it can, but the issue is for the column which does not exist in user location table(representing location capture) and is present in user action table (representing userAction). example includes, beneficiaryTag, resourceTag etc.

Reply:
so if we want one row mapper to serve both cases, it will become complex in operation and complexity may increase when there are more use cases for user action (polio transit fixed post etc)